mls-rs/src/external_client/group.rs

```diff
@@ -638,6 +638,14 @@ where
         Ok(EventOrContent::Content(auth_content))
     }
 
+    #[cfg(all(feature = "export_key_generation", feature = "private_message"))]
+    async fn get_unauthenticated_key_generation_from_sender_data(
+        &mut self,
+        _cipher_text: &PrivateMessage,
+    ) -> Result<Option<u32>, MlsError> {
+        Ok(None)
```

Comment:
What are your thoughts on `Option<u32>` instead of `u32?` I thought `Option` encapsulated this external_client situation better, since it's not a MlsError, but I'm also not 100% convinced.